binding_generator.go

```diff
@@ -0,0 +1,23 @@
+package flixkit
+
+import (
+	"fmt"
+	"log"
+)
+
+func Generate(lang string, flix *FlowInteractionTemplate, templateLocation string, isLocal bool) (string, error) {
```

Comment:
In general i prefer to use an interface for a generator and implement that for each language rather then having this dispatch function, that way we do not need to mutate this code when we add a new generator type.